### PR TITLE
Add captive core packages

### PR DESCRIPTION
**Pull request by jcooter**
This is related to stellar/ops#534

### CODE REVIEW

File: stellar-captive-core/debian/control [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+Section: web
+Priority: optional
+Maintainer: Package Maintainer <packages@stellar.org>
+Build-Depends: debhelper (>=9)
+Standards-Version: 3.9.6
+Homepage: https://www.stellar.org/
+
+Package: stellar-captive-core
+Architecture: any
+Depends: stellar-core-postgres, stellar-horizon
```
Comment (jacekn) [inline stellar-captive-core/debian/control:11]: Is this right? Does captive-core need PG?

File: stellar-captive-core-api/debian/control [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Source: stellar-captive-core-api
+Section: web
+Priority: optional
+Maintainer: Package Maintainer <packages@stellar.org>
+Build-Depends: debhelper (>=9), dh-systemd (>=1.5)
+Standards-Version: 3.9.6
+Homepage: https://www.stellar.org/
+
+Package: stellar-captive-core-api
+Architecture: any
```
Comment (jacekn) [inline stellar-captive-core-api/debian/control:10]: Can API work without core? If not they I think we want to add dependency here

File: stellar-captive-core/debian/stellar-captive-core.defaults [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+FRIENDBOT_SECRET=
+PORT=8000
+SENTRY_DSN=
+LOGGLY_TOKEN=
+PER_HOUR_RATE_LIMIT=9000
+INGEST=true
+HISTORY_ARCHIVE_URLS="https://history.stellar.org/prd/core-testnet/core_testnet_001"
+INTERNAL_PORT=4555
+# ingestion is currently only supported on 1 Horizon instance
```
Comment (jacekn) [inline stellar-captive-core/debian/stellar-captive-core.defaults:21]: I think this is no longer true?

**Review by jacekn — COMMENTED** [review]
I added small comments

File: stellar-captive-core/debian/control (hunk 1 above)
Comment (jcooter) [reply stellar-captive-core/debian/control:11]: captive-core doesn't, but stellar-core (which it depends on) does.  the stellar-core-postgres package is a virtual packages that configures stellar-core and postgres on the same box

File: stellar-captive-core/debian/stellar-captive-core.defaults (hunk 3 above)
Comment (jcooter) [reply stellar-captive-core/debian/stellar-captive-core.defaults:21]: I'm not sure.  Based on the documentation with the captive core API package captive core appears to be only able to feed one horizon instance, but I'm not sure if that extends to only one horizon instance being able to ingest.  @brahman81 any insight?

Comment (jcooter) [reply stellar-captive-core/debian/stellar-captive-core.defaults]: per discussion during standup this comment is misleading and I've removed it

**Comment by bartekn** [comment]
Added @tamirms as a reviewer because he knows `captive-core` best.

File: stellar-captive-core/debian/control (hunk 1 above)
Comment (bartekn) [reply stellar-captive-core/debian/control:11]: Stellar-Core running in captive mode does not need access to a DB, it stores all the data in memory. So I think we don't need postgres. @tamirms can confirm.

File: stellar-captive-core/debian/stellar-captive-core.defaults [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+# end Captive Core
+
+FRIENDBOT_SECRET=
+PORT=8000
+SENTRY_DSN=
+LOGGLY_TOKEN=
+PER_HOUR_RATE_LIMIT=9000
+INGEST=true
+HISTORY_ARCHIVE_URLS="https://history.stellar.org/prd/core-testnet/core_testnet_001"
+INTERNAL_PORT=4555
```
Comment (bartekn) [inline stellar-captive-core/debian/stellar-captive-core.defaults:20]: I think most of the values here are not needed. I think the ones we need are listed [here](https://github.com/stellar/go/blob/master/exp/services/captivecore/README.md#usage). @tamirms to confirm.

File: stellar-captive-core/debian/control (hunk 1 above)
Comment (tamirms) [reply stellar-captive-core/debian/control:11]: @bartekn is right, the DB is not necessary

File: stellar-captive-core/debian/stellar-captive-core.defaults (hunk 4 above)
Comment (tamirms) [reply stellar-captive-core/debian/stellar-captive-core.defaults:20]: @bartekn I believe this configuration is *not* for the captive core api. The defaults for the API are in stellar-captive-core-api/debian/stellar-captive-core-api.default . This file defines the defaults for the captive-core debian package which is essentially horizon bundled with captive core as a subprocess. So all these configuration values should match the horizon configuration values

**Review by jacekn — COMMENTED** [review]
Seems fine to me but I'll leave final approval to @brahman81

**Review by brahman81 — APPROVED** [review]
LGTM, this is a great start! 

As discussed we might want to deploy a sample pubnet horizon config under /etc/stellar dir and as @jacekn mentioned we should add a comment in the testnet version of /etc/default/stellar-horizon file pointing to pubnet version

Next step will be to build out the unstable packages to run some testing and then update documentation before releasing to stable repo.

Nice work @jcooter !

**Pull request merged**